Src/examples/cURL/smtp_mime/application.e

```diff
@@ -0,0 +1,528 @@
+﻿note
+	description	: "[
+			cURL https://curl.haxx.se/libcurl/c/smtp-mail.html
+			For original C version, please see:
+			https://curl.haxx.se/libcurl/c/smtp-mime.html
+			This is a simple example showing how to send mime mail using libcurl's SMTP
+ 			capabilities
+ 			]"
+	status: "See notice at end of class."
+	legal: "See notice at end of class."
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=cURL smtp mime example", "src=https://curl.haxx.se/libcurl/c/smtp-mime.html", "protocol=uri"
+
+class
+	APPLICATION
+
+create
+	make
+
+feature {NONE} -- Initialization
+
+	make
+			-- Run application.
+		local
+			l_result: INTEGER
+			l_from: STRING
+			l_to: STRING
+			l_cc: STRING
+			recipients: POINTER
+			headers: POINTER
+			slists: POINTER
+			curl_externals: CURL_EXTERNALS
+			l_message: STRING
+			mime: POINTER
+			alt: POINTER
+			part: POINTER
+		do
+			io.put_string ("Eiffel cURL SMTP example showing how to send e-mails with mime.")
+			io.put_new_line
+			print ("%NLibCurl Veriosn:" + (create {CURL_OPT_CONSTANTS}).libcurl_version.out)
+
+				-- add the from email.
+			l_from := "sender@example.org"
+			l_to := "addressee@example.net"
+			l_cc := "info@example.org"
+
+			create l_message.make_from_string (message)
+			l_message.replace_substring_all ("$TO", l_to)
+			l_message.replace_substring_all ("$CC", l_cc)
+			l_message.replace_substring_all ("$FROM", l_from)
+			if curl_easy.is_dynamic_library_exists then
```

Comment:
each time I see this `is_dynamic_library_exists`  I find the name bad .. should be `dynamic_library_exists` or even `dynamic_library_available` 
but I guess too late to change.